src/ActionList/Divider.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import styled from 'styled-components'
+import {get} from '../constants'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface DividerProps {}
+
+const StyledDivider = styled.div`
+  position: relative;
+  height: 1px;
+  background: ${get('colors.selectMenu.borderSecondary')};
+  border: 0;
+  margin: calc(${get('space.2')} - 1px) -${get('space.2')} ${get('space.2')};
+  padding: 0;
+
+  ::before {
+    content: '';
+    display: block;
+    position: absolute;
+    width: calc(100% - ${get('space.3')});
+    height: 1px;
+    background: ${get('colors.selectMenu.borderSecondary')};
+    top: 0;
+    left: ${get('space.2')};
+  }
+`
+
+export function Divider(props?: DividerProps): JSX.Element {
+  return <StyledDivider data-component="ActionList.Divider" {...props} />
```

Comment:
I noticed that many of the components have this. What is it for? Do other components add this?

Reply:
I added it so I could quickly determine which component matched markup in the “Elements” tab in browser devtools (i.e. not React devtools, which uses [`displayName`](https://reactjs.org/docs/react-component.html#displayname) analogously).

Is there a more idiomatic way to accomplish this? If not, I’m happy to conditionally apply the props in non-production environments, or remove them entirely.

Reply:
Got it. I don't necessarily have a problem with it, although I guess my preference would be to remove it unless it's needed (e.g. for automated tests). I feel like the conditional application would make the code less readable for such a marginal benefit.

Reply:
Removed in https://github.com/primer/components/pull/1078/commits/78ed7353bcefda20998b5a53e802e9504d3e76bb!